### ISSUE TITLE
(Android) Diablo 2r no longer boots on GitHub 213

### ISSUE THREAD

**Issue by Siber009** [yuzu-emu/yuzu#12816]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

GitHub 213

### Description of Issue

Game fails to boot just stops on a black screen. This happens on both system and turnip 

### Expected Behavior

Game runs

### Reproduction Steps

Launch game 

### Log File

[d2r 213yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/14071027/d2r.213yuzu_log.txt)

### System Configuration

CPU:8 gen 2
GPU/Driver: system and r16
RAM: 8
OS: one UI 6

**Comment by liamwhite** [comment]
Can you provide a log from the most recent working version (212?)

**Comment by Siber009** [comment]
@liamwhite here's one from 212 I tried to cut it off short because of all the networks spam that happens

[Uploading D2R on 212yuzu_log-3.txt…]()

**Comment by Siber009** [comment]
[D2R on 212yuzu_log-3.txt](https://github.com/yuzu-emu/yuzu/files/14071127/D2R.on.212yuzu_log-3.txt)

**Comment by liamwhite** [comment]
will be fixed in next build by https://github.com/yuzu-emu/yuzu/compare/30fd7aa571198c623be86df62c526abc9fdd86ba..9d01af5c9d3c378db2f516b5e1f156e2c73754d9

**Comment by Siber009** [comment]
@liamwhite amazing thank you for your hard works as always!

 As well another bug report I submitted didn't get much attention I was wondering if you knew how to fix the textures in this game just turning into rainbow bricks. It's the only thing stopping this game from being fully playable in classic mode.

I had to post the log in discord as it's to big to be uploaded here and I'm unable to upload a compressed version 
https://discord.com/channels/398318088170242053/1113238919304204359/1200810683470721024
